storage/container_storage_interface/persistent-storage-csi-ovirt.adoc

```diff
@@ -7,11 +7,11 @@ toc::[]
 
 == Overview
 
-{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for {rh-virtualization-first} ({rh-virtualization}).
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for {rh-virtualization-first}.
 
 Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
 
-To create CSI-provisioned PVs that mount to {rh-virtualization-first} ({rh-virtualization}) storage assets, {product-title} installs the oVirt CSI Driver Operator and the oVirt CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+To create CSI-provisioned PVs that mount to {rh-virtualization-first} storage assets, {product-title} installs the oVirt CSI Driver Operator and the oVirt CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
```

Comment:
You could replace `{rh-virtualization-first}` with `{rh-virtualization})` in this line, because the RHV acronym has already been defined in the first paragraph. I.e. this sentence could read as follows, once built:

"To create CSI-provisioned PVs that mount to RHV".

Or, is the convention for RHV to declare the full product name and the acronym each time?